core/src/main/java/org/hisp/dhis/android/core/program/ProgramCall.java

```diff
@@ -208,7 +209,8 @@ public boolean isExecuted() {
                                         DataElement.deleted, DataElement.dimension, DataElement.displayFormName,
                                         DataElement.domainType, DataElement.formName, DataElement.numberType,
                                         DataElement.valueType, DataElement.zeroIsSignificant,
-                                        DataElement.optionSet.with(OptionSet.uid, OptionSet.version)
+                                        DataElement.optionSet.with(OptionSet.uid, OptionSet.version),
+                                        DataElement.categoryCombo.with(CategoryCombo.uid)
```

Comment:
I think we don't need to get the categorycombos here. Dataelements belonging to a program don't have categorycombos.